l10n_br_fiscal/data/operation_data.xml

```diff
@@ -83,6 +84,36 @@
         <field name="state">approved</field>
     </record>
 
+    <record id="fo_venda_final" model="l10n_br_fiscal.operation">
+        <field name="code">Venda Consumo Final</field>
+        <field name="name">Venda Consumo Final</field>
+        <field name="fiscal_operation_type">out</field>
+        <field name="fiscal_type">sale</field>
+        <field name="default_price_unit">sale_price</field>
+        <field name="ind_final">1</field>
+        <field name="state">approved</field>
+    </record>
+
+    <record id="fo_venda_final_venda" model="l10n_br_fiscal.operation.line">
```

Comment:
Why the reason for split Sale and Sale final consumption? Today we have Sale operation and many operation line for final consumption and non-consumption. This type of change only makes complex parameterization unnecessary and makes it difficult to view reports...

Reply:
The `ind_final` field only exists in the `l10n_br_fiscal.operation` model and not in the `l10n_br_fiscal.operation.line`. In the taxes calculation, the parameter used for making the decision of final consuption is the operation field `ind_final`. That's why we needed to add this extra duplicate sale operation to make the system able to calculate the correct taxes. I agree that it should be as simple as possible for the final user, but this would require a change in the taxes calculation. More specifically here https://github.com/OCA/l10n-brazil/blob/12.0/l10n_br_fiscal/models/tax.py#L362.

Reply:
Guys could you check what we had done on v10:

https://github.com/kmee/l10n-brazil/blob/10.0-develop/sped_imposto/models/sped_operacao.py#L210-L214
https://github.com/kmee/l10n-brazil/blob/10.0-develop/sped_imposto/models/sped_operacao_item.py#L47-L51